velociraptor/catalogue/registration.py

```diff
@@ -862,30 +862,61 @@ def registration_number(
     return unit, full_name, snake_case
 
 
-def registration_gas_species_masses(
+def registration_gas_H_and_He_masses(
     field_path: str, unit_system: VelociraptorUnits
 ) -> (unyt.Unit, str, str):
     """
-    Registers the HI masses in apertures.
+    Registers the masses in Hydrogen & Helium within apertures
     """
 
     unit = unit_system.mass
 
     # Capture aperture size
-    match_string = (
-        "Aperture_([a-zA-Z]*)_index_0_aperture_total_gas_([0-9]*)_kpc"
-    )
+    match_string = "Aperture_([a-zA-Z]*)_aperture_total_gas_([0-9]*)_kpc"
     regex = cached_regex(match_string)
 
     match = regex.match(field_path)
 
     if match:
         long_species = match.group(1)
         aperture_size = match.group(2)
-    
+
+        try:
+            short_species = {"HeliumMasses": "He", "HydrogenMasses": "H"}[long_species]
+        except KeyError:
+            raise RegistrationDoesNotMatchError
+
+        full_name = f"{short_species} gas mass ({aperture_size} kpc)"
```

Comment:
Can we use the names of the elements?

```
Hydrogen Gas Mass $M_{\\rm H}$ (100 kpc)
```

Can still have `H_mass_100_kpc` for snake_case.

Reply:
P.S. we capitalise the names given.